db-engine/src/main/java/com/splicemachine/db/impl/sql/catalog/DataDictionaryCache.java

```diff
@@ -128,6 +129,7 @@ public void onRemoval(RemovalNotification<Object, Dependent> removalNotification
         aliasDescriptorCache = new ManagedCache<>(CacheBuilder.newBuilder().recordStats().maximumSize(1024).build());
         roleCache = new ManagedCache<>(CacheBuilder.newBuilder().recordStats().maximumSize(100).build());
         permissionsCache=new ManagedCache<>(CacheBuilder.newBuilder().recordStats().maximumSize(permissionsCacheSize).build());
+        defaultRoleCache = new ManagedCache<>(CacheBuilder.newBuilder().recordStats().maximumSize(1024).build());
         this.dd = dd;
```

Comment:
Maybe not in the scope of this PR. But can we have the sizes of various (dictionary) caches configurable?